extension/src/test/e2e/util.ts

```diff
@@ -1,10 +1,10 @@
 import {
   ChainablePromiseArray,
   ChainablePromiseElement,
-  ElementArray
+  ElementArray,
+  Key
 } from 'webdriverio'
 import { ViewControl } from 'wdio-vscode-service'
-import { delay } from '../../util/time'
```

Comment:
[F] Good thing the new v8 API has it's own delay as we can't import anything from our own repo here (not ESM).